framework/justfile

```diff
@@ -54,6 +54,7 @@ migrate +CHAINS:
   just run-script migrate {{CHAINS}}
 
 publish-schemas version:
+  mkdir -p ../../schemas
```

Comment:
We don't want a script to create a dir that's outside of the repo right?

Reply:
Right